pmda/rdf.py

```diff
@@ -163,3 +160,14 @@ def _conclude(self, ):
 
         rdf = self.count / (density * vol * self.nf)
         self.rdf = rdf
+
+    def _reduce(self, res, result_single_frame):
+        """ 'add' action for an accumulator"""
+        if res == []:
```

Comment:
The pythonic way is to declare the variable as `None`.  My preferred way is to pre calculate the size of `res` and initialize it as `res = np.zeros(shape)`. This way the if-else clause isn't needed.

Reply:
Your problem is that `res` is initialized in ParallelBase right?